src/views/organisations/Edit.vue

```diff
@@ -88,16 +92,27 @@ export default {
           if (data.description === this.organisation.description) {
             delete data.description;
           }
-          if (data.url === this.organisation.url) {
+          if (data.url === this.organisation.url || '') {
```

Comment:
We need to wrap the logical "or" operator in parenthesis, otherwise, JS interprets this as:
```js
if ((data.url === this.organisation.url) || '') { /*...*/ }
```

```suggestion
          if (data.url === (this.organisation.url || '')) {
```

Reply:
Done. Sorry stupid mistake.